lib/errorHandlers/suppressError.ts

```diff
@@ -42,17 +40,24 @@ function createPlatformVersionError(
   }
 
   uiLine();
-  logger.error(i18n(`${i18nKey}.platformVersionErrors.header`));
+  logger.error(
+    i18n(`lib.errorHandlers.suppressErrors.platformVersionErrors.header`)
+  );
   logger.log(
-    i18n(`${i18nKey}.platformVersionErrors.${translationKey}`, {
-      platformVersion,
-    })
+    i18n(
+      `lib.errorHandlers.suppressErrors.platformVersionErrors.${translationKey}`,
```

Comment:
This `translationKey` needs to be deleted, correct?

Reply:
I left this alone because it is dynamic.  When we switch over to the new lang file, we can refactor it to be removed.  For this scope of this I was just trying to inline as much as possible